python/vimsence.py

```diff
@@ -36,7 +36,7 @@
 has_thumbnail = {
     'c', 'cr', 'hs', 'json', 'nim', 'ruby', 'cpp', 'go', 'javascript', 'markdown',
     'typescript', 'python', 'vim', 'rust', 'css', 'html', 'vue', 'paco', 'tex', 'sh',
-    'elixir', 'cs'
+    'elixir', 'cs', 'ocaml
```

Comment:
You forgot a closing quote.

```suggestion
    'elixir', 'cs', 'ocaml'
```